ScreenBoom/ScreenBoom/ViewModel/EventDetailViewModel.swift

```diff
@@ -38,25 +38,29 @@ class EventDetailViewModel: NSObject {
   }
   
   // Add Event Detail
-  func addEventDetail(event :Event , eventdetail: EventDetail, completion:(@escaping(Result<String>) -> Void)) {
+  func addEventDetail(event :Event , eventdetail: EventDetail, completion:(@escaping(Result<Void>) -> Void)) {
     
     let eventFIRReferance = Database.database().reference().child(firebaseNodeNames.eventDetailNode).child(event.eventName)
     
     var eventDetails = [String:String]()
+    
     switch event.eventType {
```

Comment:
Can we add some comments here to explain what each of these is doing and how the event detail is being set up ?